IPs/RSKIP383.md

```diff
@@ -0,0 +1,64 @@
+---
+rskip: 383
+title: POWpeg migration age normalization
+created: 10-MAY-23
+author: JD
+purpose: Fair
+layer: Core
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          |383           |
+| :------------ |:-------------|
+|**Title**      |POWpeg migration age normalization |
+|**Created**    |10-MAY-23 |
+|**Author**     |JD |
+|**Purpose**    |Fair |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+Everytime the POWpeg composition changes a migration takes place. This migration consists of three stages:
+- Preparation: A new POWpeg has been voted but the new POWpeg has no control yet.
+- Activation: The new POWpeg is now active but the funds from the retiring POWpeg have not started to migrate yet.
```

Comment:
I'm not sure the activation is a `stage`, it's something that happens in an instant.

Perhaps the three stages could be
- Pre-activation (or pending)?
- Migration
- Standard? (Not sure about this name, couldn't think of anything better)

What do you think?

Reply:
I'm renaming the stages as follows:
- Pre-activation
- Pre-migration
- Migration

Reply:
New renaming looks good to me